packages/nx-plugin/src/generators/create-package/files/create-framework-package/bin/index.ts__tmpl__

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
+
+import { createWorkspace } from 'create-nx-workspace';
+
+async function main() {
+  const name = process.argv[2]; // TODO: use libraries like yargs or enquirer to set your workspace name
+  // TODO: update below to customize the workspace
+  await createWorkspace('<%= preset %>', {
```

Comment:
followup: Figure out how to link this project to the plugin.. without adding it to this package.json